client/packages/system/src/Patient/VaccinationCards/DetailView/DetailView.tsx

```diff
@@ -45,32 +77,35 @@ export const VaccinationCardComponent: FC = () => {
     {
       key: 'status',
       label: 'label.status',
-      accessor: ({ rowData }) => (rowData?.status ? t(rowData?.status) : ''),
-      // Cell: DotCell,
+      accessor: ({ rowData }) => rowData?.status,
+      Cell: ({ ...props }) => (
+        <StatusCell
+          {...props}
+          statusMap={{
+            'label.status-given': {
+              color: theme.palette.success.light,
+              label: t('label.status-given'),
+            },
+            'label.status-not-given': {
+              color: theme.palette.error.main,
+              label: t('label.status-not-given'),
+            },
+          }}
```

Comment:
Presumably the status values will be back-end enums?

Reply:
Yeah great point - agree it probably should be! 

Especially once we implement the age ranges - then we should implement a `Late` status, which comes into effect once the suggested date has passed, and the patient is beyond the end of the age range 👍

Reply:
For now I've just used the `given` value (true/false) as the status map key. Can be easily changed to enum values at some point.

7ed8975781